.vib/grafana-tempo/vib-publish.json

```diff
@@ -90,4 +90,4 @@
       ]
     }
   }
-}
+}
```

Comment:
I missed this in my previous review, but you are missing the `ginkgo` action in both VIB pipelines.

Reply:
Yes, I missed that copying and pasting the raw file from the previous commit. It is added now for both pipelines.